lib/plines/step.rb

```diff
@@ -65,6 +65,10 @@ def fan_out(&block)
       fan_out_blocks << block
     end
 
+    def run_if?(&block)
```

Comment:
The ruby convention is to only use `?` on the end of predicates -- methods that answer a question with a true or false.  This method is not a predicate -- instead it produces the side effect of defining a fan out, where the passed `block` should be a predicate.  So this should be `run_if`, not `run_if?` to follow the ruby convention and have `?` retain its normal meaning.

Also, I expect I would be more likely to use `skip_if` than `run_if`.  In my experience, most jobs in a pipeline are normally included and it's the exception that it's skipped rather than the exception that it's run.  WDYT about defining both `run_if` and `skip_if`?